### DISCUSSION TITLE
Getting started Guide?

### DISCUSSION THREAD

**Discussion by hunter-goller** [community/community#6624] [Copilot Conversations] [answered]
Is there a getting started guide? I got the extension installed, but the guide URL just brings up an error 404 page not found.

**Comment by MohamedElashri** [comment]
> Is there a getting started guide? I got the extension installed, but the guide URL just brings up an error 404 page not found.

yes, that's the [link](https://github.com/github/copilot-preview/blob/main/docs/gettingstartedvisualstudiocode.md)

**Reply by ShoGinn** [reply]
This link has a 404 as well.... (I think this is because something is wrong with getting added to the repo..)

**Reply by MohamedElashri** [reply]
Maybe it is a problem with access, It might happen that you lost access to this private repository

**Reply by ShoGinn** [reply]
> Maybe it is a problem with access, It might happen that you lost access to this private repository

I was just added to the preview.. so like @kRoll242 I think I'll just wait 24hrs or so ;)

**Comment by kRoll242** [comment]
+1 I was added today but the links are 404.  However I can see it is enabled for my account.
![image](https://user-images.githubusercontent.com/3271310/138525812-9dbf63a5-6b95-420c-b64c-46d268b19278.png)

**Reply by aicrumb** [reply]
same thing over here

**Reply by Victorsitou** [reply]
I had the same issue, try creating a ticket [here](https://support.github.com/contact?tags=docs-generic).

**Comment by orph** [comment, marked as answer]
The underlying technical issue leading to limited access is now fixed. So sorry about that. Please try again and report back if you have any trouble accessing.
